src/Handler/MockHandler.php

```diff
@@ -87,6 +87,19 @@ function ($value) use ($request, $options) {
                 if ($this->onFulfilled) {
                     call_user_func($this->onFulfilled, $value);
                 }
+                if (isset($options['sink'])) {
+                    $contents = (string) $value->getBody();
+                    $sink = $options['sink'];
+
+                    if (is_resource($sink)) {
```

Comment:
PHP tmp stream only create a temp file when the stream exceeds a certain size (2 megs by default I believe). Are you saying that you're seeing a temp file created no matter what or that you had to manually create a temp file to make what I suggested work?